### ISSUE TITLE
Ability to define a custom namespace for external helpers

### ISSUE THREAD

**Issue by buzinas** [babel/babel#5217]
Currently the [code sets up a hard-coded namespace `babelHelpers`](https://github.com/babel/babel/blob/master/packages/babel-plugin-external-helpers/src/index.js).

I would like to have the ability to define my own name, eg:
```js
{
  "plugins": [
    ["external-helpers": { "namespace": "_myNamespace" }]
  ]
}
```

```javascript
export default function ({ types: t }) {
  return {
    pre(file) {
      file.set("helpersNamespace", t.identifier(file.opts.namespace || "babelHelpers"));
    }
  };
}
```
Do you see any problem if we add this option to the external-helpers plugin?

**Comment by hzoo** [comment]
Thanks for bringing this up - curious what's your use case for doing this?

**Comment by buzinas** [comment]
@hzoo basically it's because we have a very big application, and we're trying to reduce as most as we can the footprint. We're already injecting only the external helpers we actually use, and minifying them, but we have ~8kb of the word "babelHelpers" in our final JS file.

Of course gzipping helps there, but want to change that to any 2-letters word, and it would be simple if we could customize that. Currently, we copied that plugin internally and hardcoded another namespace, but it would be good if the plugin offered this possibility.

**Comment by hzoo** [comment]
So gzip/minify `babelHelpers` doesn't do anything since it's global?

**Comment by buzinas** [comment]
Yeah, that's right. But found a way to use Webpack Provide plugin to inject babelHelpers into each module, and then UglifyJS could rename it :)

Although, since it's a pretty simple change, it could help other people that don't use Webpack.

**Comment by rafayepes** [comment]
@buzinas do you have an example setup?
